postgres/src/rows.rs

```diff
@@ -259,13 +256,21 @@ impl<'a> Row<'a> {
 pub trait RowIndex {
     /// Returns the index of the appropriate column, or `None` if no such
     /// column exists.
-    fn idx(&self, stmt: &Statement) -> Option<usize>;
+    fn idx<I: AsRef<StatementInfo>>(&self, info: &I) -> Option<usize> {
```

Comment:
Oh, bummer. I didn't realize RowIndex was actually intended to be part of the public API.

At the point this is a breaking change, should we just drop the lifetime parameter from `Rows`?